requirements.txt

```diff
@@ -29,3 +29,4 @@ psutil==5.6.6
 cryptography>=2.5,<3
 Click>=7.0,<7.1
 cbor>=1.0.0
+cysecuretools>=2.0.0,<=2.1.0
```

Comment:
Travis CI failed for Python 2.7 and 3.5 because this package depends on python 3.6 or above.

Reply:
@ARMmbed/mbed-os-tools 
More and more packages are diverging from older versions of Python. Does `requirements.txt` in Mbed OS need backward compatibility at this point?